docs/_template.html

```diff
@@ -0,0 +1,83 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+
+<head>
```

Comment:
Currently, the right font isn't displaying.
You will need to add
```
    <link rel="preconnect" href="https://fonts.googleapis.com">
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
    <link href="https://fonts.googleapis.com/css2?family=Reem+Kufi:wght@400;500;700&display=swap" rel="stylesheet">
```
at some point I believe.